<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFind</class>
 <widget class="QWidget" name="FormFind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="placeholderText">
          <string>账号/昵称/手机号码/邮箱</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFind">
         <property name="styleSheet">
          <string notr="true">color:white;background-color:rgb(0,155,219)</string>
         </property>
         <property name="text">
          <string>查找</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
